modules/claims_api/lib/claims_api/evss_bgs_mapper.rb

```diff
@@ -46,6 +46,16 @@ def map_and_build_object # rubocop:disable Metrics/MethodLength
 
     private
 
+    def contentions
+      if @data['contentions'].nil?
+        []
+      elsif @data['contentions'].is_a?(String)
+        @data['contentions'].split(', ')
```

Comment:
these don't always work this well. we can end up with the same "contention" with a comma nested inside of it. I think there's a regex pattern in v2 that will handle this though.